repos/spack_repo/builtin/packages/geant4/package.py

```diff
@@ -198,7 +202,8 @@ class Geant4(CMakePackage):
             depends_on("qt-base +accessibility +gui +opengl")
         with when("^[virtuals=qmake] qt"):
             depends_on("qt@5: +opengl")
-            depends_on("qt@5.9:", when="@11.2:")
+            depends_on("qt@5.9:", when="@11.2:11.3")
+            depends_on("qt@6:", when="@11.4:")
```

Comment:
Package renamed. See above: `qt-base`. I'd replace this with
```suggestion
            conflicts("^qt", when="@11.4:")
```

Reply:
Or add a conflicts two lines below with the "Qt6 not supported before 11.2" conflict. That's probably cleaner.